org.eclipse.draw2d/src/org/eclipse/draw2d/Viewport.java

```diff
@@ -27,7 +27,10 @@ public class Viewport extends Figure implements PropertyChangeListener {
 
 	/** ID for the view location property */
 	public static final String PROPERTY_VIEW_LOCATION = "viewLocation"; //$NON-NLS-1$
-	private IFigure view;
+	/**
+	 * @since 3.21
+	 */
+	protected IFigure view;
```

Comment:
I had to make view somehow accessible for the subclass FreeformViewport so that the layout there works. This does not feel good.